packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div id="oc-files-sharing-sidebar" class="oc-position-relative">
+    <invite-collaborator-form v-if="currentUserCanShare" key="new-collaborator" class="oc-my-s" />
+    <p
+      v-else
+      key="no-reshare-permissions-message"
+      data-testid="files-collaborators-no-reshare-permissions-message"
+      v-text="noResharePermsMessage"
+    />
     <oc-loader v-if="sharesLoading" :aria-label="$gettext('Loading people list')" />
     <template v-else>
-      <invite-collaborator-form v-if="currentUserCanShare" key="new-collaborator" class="oc-my-s" />
-      <p
-        v-else
-        key="no-reshare-permissions-message"
-        data-testid="files-collaborators-no-reshare-permissions-message"
-        v-text="noResharePermsMessage"
-      />
       <div v-if="hasSharees" class="avatars-wrapper oc-flex oc-flex-middle oc-flex-between">
```

Comment:
this change fixes some intermittent fails in e2e such as this issue https://github.com/owncloud/web/issues/6596#issuecomment-1068722941

Reply:
yes, but this should not be the solutions..... it's a nasty behavior by web :( i have to think how to fix that best.

Reply:
@saw-jan fixed it, lets see if the ci likes it ;)